integration/integration_test.go

```diff
@@ -4985,22 +4891,24 @@ func (s *IntSuite) TestExecEvents(c *check.C) {
 	}
 
 	for _, tt := range execTests {
-		// Create client for each test in grid tests
-		clientConfig := ClientConfig{
-			Login:       s.me.Username,
-			Cluster:     Site,
-			Host:        Host,
-			Port:        main.GetPortSSHInt(),
-			Interactive: tt.isInteractive,
-		}
-		name := check.Commentf(tt.name)
-		_, err := runCommand(main, []string{lsPath}, clientConfig, 1)
-		c.Assert(err, check.IsNil, name)
-		// Make sure the exec event was emitted to the audit log.
-		eventFields, err := findEventInLog(main, events.ExecEvent)
-		c.Assert(err, check.IsNil, name)
-		c.Assert(eventFields.GetCode(), check.Equals, events.ExecCode, name)
-		c.Assert(eventFields.GetString(events.ExecEventCommand), check.Equals, tt.outCommand, name)
+		t.Run("", func(t *testing.T) {
```

Comment:
Nit: Can you add description?

Reply:
Done

Reply:
Done, and shortened to remove stuttering in test name